spanner/spansql/parser.go

```diff
@@ -2111,6 +2111,13 @@ func (p *parser) parseSelectFrom() (SelectFrom, *parseError) {
 		return nil, err
 	}
 	sf := SelectFromTable{Table: tname}
+	if p.eat("@") {
+		hint, err := p.parseHints(map[string]string{}, false)
```

Comment:
Hmm... It seems that the [documentation](https://cloud.google.com/spanner/docs/query-syntax#from_clause) is slightly off here. It is actually possible to have more than one table hint, so the variable here should be renamed to `hints` and the `parseHints(..)` function should be called with `true` rather than `false`.

For example the following query is accepted by Cloud Spanner:

```sql
select *
from albums@{FORCE_INDEX=_BASE_TABLE, GROUPBY_SCAN_OPTIMIZATION=FALSE}
```